package.json

```diff
@@ -131,7 +131,7 @@
     "test-prod-build": "yarn test --deprecated 'yarn test --prod --build'",
     "test-build": "yarn test --deprecated 'yarn test --build'",
     "test-build-prod": "yarn test --deprecated 'yarn test --build --prod'",
-    "test-build-devtools": "yarn test --build --project devtools",
+    "test-build-devtools": "node ./scripts/jest/jest-cli.js --build --project devtools --release-channel=experimental",
```

Comment:
[CI is using experimental](https://app.circleci.com/pipelines/github/facebook/react/24848/workflows/f4e05c41-aad9-4780-bdbd-f9335d89b515/jobs/451835)

Previous command would use www-modern

Reply:
Also @rickhanlonii might want to chime in since he was working on these test commands recently.